tsconfig.json

```diff
@@ -0,0 +1,72 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
```

Comment:
why so many comments?

Reply:
This is autogenerated, I have to clean it up, but I decided to keep it until I better understand what are the options that we really need.

Reply:
you can use the one that we already have in pe-js